### ISSUE TITLE
Vite v5 compatibility?

### ISSUE THREAD

**Issue by mattbrictson** [cmalven/vite-plugin-sass-glob-import#11]
When installing this package in a project using Vite v5, I get the following warning:

```
warning " > vite-plugin-sass-glob-import@2.0.0" has incorrect peer dependency "vite@^3.0.0 || ^4.0.0".
```

The plugin still seems to work, however.

Are there plans to officially support Vite v5 by updating the peer dependency specification?

**Comment by cmalven** [comment]
Hey @mattbrictson yep definitely plan on updating to officially support Vite v5. It most definitely won't need any updates other than the peer dependency you mentioned. Thanks for the reminder.

**Comment by cmalven** [comment]
Release `3.0.0` which adds support for Vite v5. Please let me know if you experience any issues. https://github.com/cmalven/vite-plugin-sass-glob-import/releases/tag/3.0.0

**Comment by cmalven** [comment]
@Jackky90 I just released 3.0.1 which will hopefully fix this issue. Would you mind trying it out and letting me know if it works for you?

**Comment by Jackky90** [comment]
> @Jackky90 I just released 3.0.2 which will hopefully fix this issue. Would you mind trying it out and letting me know if it works for you?

Will check this in next 10 hours, ok

**Comment by Jackky90** [comment]
> @Jackky90 I just released 3.0.2 which will hopefully fix this issue. Would you mind trying it out and letting me know if it works for you?

Sorry, completely forgot to check this out yesterday. Just got updated and at first sight all working as expected. I think you can close this issue now. If I find something weird I will reopen issue either create new, depends.
